lambda/.dockerignore

```diff
@@ -1,2 +1 @@
-.git/
```

Comment:
Is there a reason we want the git directory inside the Docker image / container?